src/api/schoolDetails.ts

```diff
@@ -1,10 +1,10 @@
 import useSWR from 'swr';
 import fetchData from './fetchData';
 
-export const fetchSchoolDetails = (path: string) => {
+export const fetchSchoolDetails = (path: string): Promise<any> => {
   return fetchData(path).then((res) => res.results[0]);
 };
 
-export const useSchoolDetails = (schoolId: number) => {
+export const useSchoolDetails = (schoolId: number): any => {
```

Comment:
Don't SWR provide some type definitions to use?

Reply:
Yes, but you have to add arguments, which describe the type of data (and we have another issue to do this)